examples/layers/preprocessing/random_rotataion_demo.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+   random_rotation_demo.py shows how to use the RandomRotation preprocessing layer
+   for object detection. An image is downloaded from URL, both the image and
+   bounding boxes are augmented and displayed using matplotlib.
+"""
+import matplotlib.patches as patches
+import matplotlib.pyplot as plt
+import requests
+import tensorflow as tf
+from PIL import Image
+
+from keras_cv.layers import preprocessing
+
+IMG_SIZE = (256, 256)
+BATCH_SIZE = 9
+
+
+def visualize_bounding_boxes_on_image(image, bboxes, color="r", title="visualization"):
+    fig, ax = plt.subplots()
+    ax.imshow(image)
+    for box in bboxes:
+        rect = patches.Rectangle(
+            (box[0], box[1]),
+            box[2] - box[0],
+            box[3] - box[1],
+            linewidth=1,
+            edgecolor=color,
```

Comment:
Can we iterare on a color list? just cause we have some symmetries in the demo which would have a more rapid eyes matching by diversifying the color

Reply:
Thank you @bhack. That makes sense. I have added the color list for better visualization.
![before_aug_2](https://user-images.githubusercontent.com/78194266/177620316-ccdb46a9-b155-43fd-a343-c50cd26dfe05.png)
![after_aug_2](https://user-images.githubusercontent.com/78194266/177620337-2c590ea6-77f6-42da-9c64-c60db99911e1.png)